lib/git-user.js

```diff
@@ -9,7 +9,7 @@ module.exports = function () {
     email = exec('git config --get user.email')
   } catch (e) {}
 
-  name = name && name.toString().trim()
+  name = name && JSON.stringify(name.toString().trim()).slice(1, -1)
```

Comment:
The extra quotes seems to be a quirk on the node REPL. It should work fine if you are running it in a script.